NHLBI_BioData_Catalyst/python/4_Genomic_Queries.ipynb

```diff
@@ -513,7 +517,7 @@
     "# View gene list about \"Gene_with_variant\" variable\n",
     "dictionary_search = dictionary.find(\"Gene_with_variant\").DataFrame()\n",
     "gene_list = dictionary_search.loc['Gene_with_variant', 'categoryValues']\n",
-    "print(sorted(gene_list))"
```

Comment:
In above code (lines 469-471), when running this notebook on Terra the kernel dies and is unable to successfully execute these lines. Not sure if this is a show stopper (I think this has happened before), but wanted to report.

> dictionary_entries = resource.dictionary().find(\"\")
> dict_df = dictionary_entries.DataFrame()
> genotype_vars = dict_df[dict_df[\"HpdsDataType\"]==\"info\"]

Reply:
This has been changed to only reveal the first 10 entries:
print(sorted(gene_list[1:10]))